mathics/doc/doc_entries.py

```diff
@@ -373,12 +427,14 @@ def get_tests(self) -> list:
         return self.tests
 
     def is_private(self) -> bool:
+        """the tests are private, so are not included in the documentaiton"""
```

Comment:
documentaiton -> documentation

Suggestion:  Returns True if this test is "private" not supposed to be visible as example documentation.